recipes-example/template-dbus-service/template-dbus-service_git.bb

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (C) 2019 Luxoft Sweden AB
+# SPDX-License-Identifier: MIT
+#
+
+DESCRIPTION = "Template for creating a D-Bus Service"
+HOMEPAGE = "https://github.com/Pelagicore/template-dbus-service"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+PV = "1.0+git${SRCPV}"
+PR = "r0"
+
+SRC_URI = "git://github.com:Pelagicore/template-dbus-service.git;protocol=ssh;branch=master"
+SRCREV = "f8a0e1bc987c2731b98ac69f55972ee9264ab518"
+
+inherit meson systemd
+
+DEPENDS += "glibmm gdbus-codegen-glibmm-native"
```

Comment:
hm... not rdepends on `gdbus-codegen-glibmm-native`?

Reply:
Not a runtime dependency. Only used when building.